src/Core/Domain/Cart/Command/RemoveCartRuleFromCartCommand.php

```diff
@@ -45,8 +45,10 @@ class RemoveCartRuleFromCartCommand
     private $cartRuleId;
 
     /**
-     * @param $cartId
-     * @param $cartRuleId
+     * @param int $cartId
+     * @param int $cartRuleId
+     *
+     * @throws \PrestaShop\PrestaShop\Core\Domain\Cart\Exception\CartConstraintException
```

Comment:
Is there any reason to not import the exception with a `use` here?

Reply:
Fixed